assets/csidriveroperators/aws-ebs/hypershift/mgmt/kustomization.yaml

```diff
@@ -0,0 +1,50 @@
+resources:
+  - ../../base
+namespace: ${CONTROLPLANE_NAMESPACE}
+patches:
+  - path: 02sa.patch.yaml
+    target:
+      kind: ServiceAccount
+      version: v1
+  - path: lease_role.patch.yaml
+    target:
+      kind: Role
+      version: v1
+  - path: configmap_role.patch.yaml
+    target:
+      kind: Role
+      version: v1
+  - path: hypershift_role.patch.yaml
+    target:
+      kind: Role
+      version: v1
+  - path: deployment.patch.yaml
+    target:
+      kind: Deployment
+      version: v1
+  - patch: |-
+      - op: "add"
+        path: "/spec/template/spec/containers/0/args/-"
+        value: --guest-kubeconfig=/etc/guest-kubeconfig/kubeconfig
+    target:
+      kind: Deployment
+  - patch: |- 
+      $patch: delete
+      apiVersion: operator.openshift.io/v1
+      kind: ClusterCSIDriver
+      metadata:
+        name: ebs.csi.aws.com
+  - patch: |-
+      $patch: delete
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        name: aws-ebs-csi-driver-operator-clusterrole
+  - patch: |-
+      $patch: delete
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: aws-ebs-csi-driver-operator-clusterrolebinding
```

Comment:
Would it be possible to delete all objects that have / don't have a label?
We could add something like  `storage.openshift.io/designation: control-plane` / `worker` to everything in base.

Reply:
yeah done.  Using something like - https://github.com/kubernetes-sigs/kustomize/issues/1593#issuecomment-1545727117